parrot_ai/sermon_evaluation.py

```
@@ -311,11 +309,149 @@ def r2(x: float) -> float:
             Illustrations=r2(illustrations),
             Overall_Impact_Base=r2(overall_base),
-            Overall_Impact_Adjustment=r2(adjustment),
-            Adjustment_Rationale=rationale,
             Overall_Impact=r2(overall),
         )
 
+    # ----------- Post-processing: stricter calibration -----------
+    def _apply_strict_calibration(self, scoring: SermonScoringStep2, extraction: SermonExtractionStep1) -> SermonScoringStep2:
+        """Downshift inflated scores using evidence from Step 1.
+
+        Heuristics (conservative, integer outputs 1–5):
```

Ruff (0.13.2)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)